docs/content/Segments.md

```diff
@@ -4,6 +4,8 @@ layout: doc_page
 Segments
 ========
 
+Druid segments contain data for a time interval, stored as separate columns. In addition, dimensions (string columns) have an inverted indexes associated with them for each dimension value.
+
```

Comment:
Are there two uses of "version" when talking about segments?  I've seen it used when talking about the content of segments ("their versions are too old and their data has been replaced by newer segments" from Coordinator.md) but here it seems to be talking about a segment format?  If this is so, which use of "version" is being used under Naming Convention where it talks about segment identifiers?